src/www/app/outline_server_repository/index.ts

```diff
@@ -12,105 +12,19 @@
 // See the License for the specific language governing permissions and
```

Comment:
Yep, I was actually thinking this as I did this, that you probably didn't want outline_server to be exposed, but wanted to see what you said first before going through with it and increasing PR complexity

Reply:
So the import restrictions are tricky here as they don't seem to support exclusions. (I tried to use [glob syntax](https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-restricted-paths.md#rule-details) to no avail.) Because outline_server_repository is nested within app there's no clean way to exclude it from every other file without it excluding itself. I've scoped the exclusion to only the first-level children directly in the app folder and set the target non-importable files to the specific ones we want to be private.